lib/WP_Auth0_Random_Storage.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+class WP_Auth0_Random_Storage {
```

Comment:
This was previously the `WP_Auth0_Nonce_Handler`. I added a more generic cookie name and removed the `verify` method so extending classes can implement it themselves.